app/models/diagram.py

```
@@ -322,16 +332,31 @@ def to_models_code_template(self) -> dict[str, str]:
     def to_springboot_models_template(self) -> dict[str, str]:
         source = self.get_source_class().get_name().lower()
         target = self.get_target_class().get_name()
-        rel_type = "@ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})"
-        rel_type += "\n\t@JsonIgnore"
-        join = "@JoinTable("
-        join += f'\n\t\tname = "{source.replace(" ", "_")}_{target.replace(" ", "_").lower()}",'
-        join += (
-            f'\n\t\tjoinColumns = @JoinColumn(name = "{source.replace(" ", "_")}_id"),'
+        params = []
+
+        if self.get_type() == RelationshipType.COMPOSITION:
+            params.append(
+                "cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REMOVE}"
+            )
+            params.append("orphanRemoval = true\n")
+            params_str = ",\n\t\t".join(params)
+            rel_type = f"@ManyToMany(\n\t\t{params_str})\n\t@JsonIgnore"
+        else:
+            rel_type = (
+                "@ManyToMany("
```

You should add a comment on top of the 'else' clause to indicate what the 'else' indicates; which is AGGREGATION or ASSOCIATION. This will ensure that future developers understand what this means -> maintainability issue.

Even better, you can check specifically if the type is AGGREGATION or ASSOCIATION. This also affects maintainability, as if for some reason the types aren't any of the three defined, you can raise an exception here, which will greatly ease debugging process, as we can directly tell where it went wrong.